judgels-backends/judgels-commons/judgels-fs/src/main/java/judgels/fs/FileSystem.java

```diff
@@ -28,4 +28,7 @@ default void writeToFile(Path filePath, String content) {
     default String readFromFile(Path filePath) {
         return new String(readByteArrayFromFile(filePath));
     }
+
+    void copyDirectory(Path src, Path dest);
```

Comment:
Can we move this method a bit up? Wanted to keep all the default methods to be the last.